app/models/draft-registration.ts

```diff
@@ -15,6 +19,14 @@ export default class DraftRegistrationModel extends OsfModel {
     @attr('date') datetimeInitiated!: Date;
     @attr('date') datetimeUpdated!: Date;
 
+    @attr('fixstring') title!: string;
+    @attr('fixstring') description!: string;
+    @attr('fixstring') registrationDOI!: string | null;
```

Comment:
i think we can skip `registrationDOI` -- it won't exist until after the draft is registered

`articleDOI` too -- it generally refers to something published *after* the registration, and the user can add it later